src/main/java/org/codehaus/plexus/archiver/diags/NoOpArchiver.java

```diff
@@ -414,4 +414,10 @@ public String getOverrideGroupName()
         return null;
     }
 
+    @Override
+    public void configureReproducible( Date outputTimestamp )
+    {
+
```

Comment:
Empty line should be removed as in `TrackingArchiver`.

Reply:
using the convention used in this file: there is an empty line in other methods